website_docs/getting-started.md

```diff
@@ -75,23 +75,25 @@ Now, start up your application and perform a few operations to generate tracing
 
 Lastly, open a browser and navigate to the [Jaeger UI](http://localhost:16686) or [Zipkin UI](http://localhost:9411) and search for traces related to your service, which were generated by the auto-instrumentation features of OpenTelemetry!
 
-### Achievement Unlocked: Tracing Enabled
-
-Adding tracing to a single service is a great first step and although auto-instrumentation provides quite a bit of insight on its own, OpenTelemetry provides a few more features that will allow you gain even deeper insights!
-
-[Context Propagation][context-propagation] is perhaps one of the most powerful concepts in OpenTelemetry because it will upgrade your single service trace into a _distributed trace_, which makes it possible for OpenTelemetry vendors to visualize a request from end-to-end across process and network boundaries.
-
-[Span Events][events] allow you to add a human-readable message on a span that represents "something happening" during its lifetime.
-
-[Manual Instrumentation][manual-instrumentation] will give provide you the ability to enrich your traces with domain specific data.
-
-[Automatic Instrumentation][auto-instrumentation]
```

Comment:
@cartermp - this entire page talks about automatic instrumentation and that same link is given elsewhere in the page, so it seemed odd to me to mention it again here in the list of "more things to look into". Thoughts?

Reply:
Makes sense to me.